www/docs/content/plugins/payment/paypal.md

```diff
@@ -75,6 +75,8 @@ const plugins = [
 ]
 ```
 
+The endpoint for PayPal webhook integration with your Medusa backend should be set to `{BACKEND_URL}/paypal/hooks`. Make sure to replace `{BACKEND_URL}` with the URL to your backend.
```

Comment:
Should this be set in paypal's configuration? If so, I would add this under the Prerequisites section

Reply:
Yes, no problem!